server/vulnerabilities/nvd/cve_test.go

```diff
@@ -279,9 +279,12 @@ func TestTranslateCPEToCVE(t *testing.T) {
 		safeDS := &threadSafeDSMock{Store: ds}
 
 		softwareCPEs := []fleet.SoftwareCPE{
-			{CPE: "cpe:2.3:a:google:chrome:-:*:*:*:*:*:*:*", ID: 1, SoftwareID: 1},
-			{CPE: "cpe:2.3:a:mozilla:firefox:-:*:*:*:*:*:*:*", ID: 2, SoftwareID: 2},
-			{CPE: "cpe:2.3:a:haxx:curl:-:*:*:*:*:*:*:*", ID: 3, SoftwareID: 3},
+			{CPE: "cpe:2.3:a:google:chrome:*:*:*:*:*:*:*:*", ID: 1, SoftwareID: 1},
+			{CPE: "cpe:2.3:a:mozilla:firefox:*:*:*:*:*:*:*:*", ID: 2, SoftwareID: 2},
+			{CPE: "cpe:2.3:a:haxx:curl:*:*:*:*:*:*:*:*", ID: 3, SoftwareID: 3},
```

Comment:
I don't understand where the change from `-` to `*` is coming from. Is this another change we picked up from `github.com/facebookincubator/nvdtools` during upgrade?

Reply:
Yes, it seems they fixed this bug [here](https://github.com/facebookincubator/nvdtools/commit/070f4ce499c0d02fea121024f3e3e307b20f6bda).

In the cpe 2.3 format`-` means "NA" (doesn't apply), `*` means "ANY".